pkgs/development/python-modules/hebi/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  numpy,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "hebi";
```

```suggestion
  pname = "hebi-py";
```
https://pypi.org/project/hebi-py/

> Attribute names in python-packages.nix as well as pnames should match the library's name on PyPI, but be normalized according to [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names). This means that characters should be converted to lowercase and . and _ should be replaced by a single - (foo-bar-baz instead of Foo__Bar.baz). If necessary, pname has to be given a different value within fetchPypi.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines